lib/execution_engine2/sdk/EE2Runjob.py

```diff
@@ -152,6 +153,17 @@ def _check_workspace_permissions(self, wsid):
                     f"User {self.sdkmr.user_id} doesn't have permission to run jobs in workspace {wsid}."
                 )
 
+    def _check_workspace_permissions_list(self, wsids):
+        perms = self.sdkmr.get_workspace_auth().can_write_list(wsids)
+        bad_ws = [key for key in perms.keys() if perms[key] is False]
+        if len(bad_ws) > 0:
```

Comment:
`if bad_ws` would have the same effect as `if len(bad_ws) > 0`

Reply:
Changed